extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -65,13 +70,29 @@ export class Renderer {
     });
   }
 
+  addLoadingBar() {
+    return this.ampdoc_.whenReady().then(() => {
+      if (!this.getRootElement_().querySelector(
```

Comment:
Likewise, preferrable to use `ampdoc.getBody()`

Reply:
Done